alphadia/outputtransform.py

```diff
@@ -845,6 +847,8 @@ def _build_run_stat_df(
         Dataframe containing the statistics
 
     """
+    optimization_manager_path = os.path.join(folder, "optimization_manager.pkl")
```

Comment:
could we move this logic to the Managers?

cf. also this comment: https://github.com/MannLabs/alphadia/pull/301#discussion_r1717115466